### PR TITLE
Converted test projects to run on netcoreapp2.0

### PR DESCRIPTION
**Pull request by kichalla**

### CODE REVIEW

File: build/common-testapps.props [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <TestIncludeViewCompilationTargets Condition="'$(TestIncludeViewCompilationTargets)'==''">$(MvcRazorCompileOnPublish)</TestIncludeViewCompilationTargets>
   </PropertyGroup>
 
-  <Import Project="..\src\Microsoft.AspNetCore.Mvc.Razor.ViewCompilation\build\netcoreapp1.1\Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.targets"
+  <Import Project="..\src\Microsoft.AspNetCore.Mvc.Razor.ViewCompilation\build\netcoreapp2.0\Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.targets"
```
Comment (kichalla) [inline build/common-testapps.props]: @pranavkm this is changing things under 'src'. Is this OK?

Comment (pranavkm) [inline build/common-testapps.props]: I think so. @Eilon - are tools getting bumped up to netcoreapp2.0 for the next release?

File: test/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.FunctionalTests/Infrastructure/ApplicationTestFixture.cs [hunk 2]
```diff
@@ -63,8 +63,8 @@ public virtual DeploymentParameters GetDeploymentParameters()
                 RuntimeArchitecture.x64)
             {
                 PublishApplicationBeforeDeployment = true,
-#if NETCOREAPP1_1
-                TargetFramework = "netcoreapp1.1",
+#if NETCOREAPP2_0
```
Comment (pranavkm) [inline test/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.FunctionalTests/Infrastructure/ApplicationTestFixture.cs]: Think we can remove the `ifdef`, we don't compile for another TFM here and don't plan on doing that

File: build/common-testapps.props (hunk 1 above)
Comment (Eilon) [inline build/common-testapps.props]: There's a bug preventing tools from being netcoreapp2.0 due to package restore: https://github.com/aspnet/DotNetTools/issues/281

**Pull request merged**